src/main/java/com/rcv/Tabulator.java

```diff
@@ -347,12 +356,17 @@ private boolean shouldContinueTabulating() {
     // how many candidates have already been eliminated
     int numEliminatedCandidates = candidateToRoundEliminated.keySet().size();
     // how many winners have been selected
-    int numWinners = winnerToRound.size();
+    int numWinnersDeclared = winnerToRound.size();
     // apply config setting if specified
     if (config.willContinueUntilTwoCandidatesRemain()) {
-      return numEliminatedCandidates + numWinners + 1 < config.getNumCandidates();
+      return numEliminatedCandidates + numWinnersDeclared + 1 < config.getNumCandidates();
     } else {
-      return numWinners < config.getNumberOfWinners();
+      int lastWinnerRound = winnerToRound.values().stream().mapToInt(v -> v).max().orElse(-1);
```

Comment:
please explain yourself

Reply:
This is just a way of getting the max value, or -1 if there's no max. If I don't add that `orElse`, the result is an `OptionalInt`, which is surprisingly awkward to work with. I'll add a comment.